packages/pixeloven-webpack/compiler/src/index.ts

```diff
@@ -1,8 +1,9 @@
 import { mergeOptions } from "@pixeloven-core/common";
+import {Name, Target} from "@pixeloven-core/env";
 import {
     Config,
-    webpackClientConfig,
-    webpackServerConfig,
+    getConfig,
+    shimOptions
```

Comment:
@jimmynotjim I think the first thing we want to do is update the CLI so we no longer need this shim. Then we can start adding new configuration options.

Reply:
Sounds good, I was going to work on that after I finish the copy command.